crates/bevy_ui/src/render/mod.rs

```diff
@@ -755,6 +792,29 @@ pub fn queue_uinodes(
             .reserve(extracted_uinodes.uinodes.len());
 
         for (entity, extracted_uinode) in extracted_uinodes.uinodes.iter() {
```

Comment:
As before, this iterates all extracted nodes for all extracted cameras.
Also, if there are N views, would this draw the nodes that have no `UiCamera` assigned N times?

Reply:
Fixed in https://github.com/bevyengine/bevy/pull/10559/commits/1d36e126fdf2896dc02f8904af90ce683e00c47d